src/components/Modal/VotingPowerDelegationDetail/VotingPowerDistribution.tsx

```diff
@@ -3,26 +3,37 @@ import React from 'react'
 import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
 import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
 
+import { VpDistribution } from '../../../clients/SnapshotGraphql'
+import { EMPTY_DISTRIBUTION } from '../../../hooks/useVotingPowerDistribution'
+
 import './VotingPowerDistribution.css'
 import VotingPowerDistributionPopup from './VotingPowerDistributionPopup'
 
 interface Props {
-  mana: number
-  name: number
-  land: number
-  delegated: number
+  vpDistribution?: VpDistribution | null
   className?: string
 }
 
 const getPercentage = (value: number, total: number): string => `${((value * 100) / total).toFixed(2)}%`
 
-const VotingPowerDistribution = ({ mana, name, land, delegated, className }: Props) => {
+const VotingPowerDistribution = ({ vpDistribution, className }: Props) => {
+  if (!vpDistribution) vpDistribution = EMPTY_DISTRIBUTION
```

Comment:
I'd avoid reassigning props. We can add `vpDistribution={vpDistribution || EMPTY_DESTRIBUTION}` as a prop where this component is rendered.

Reply:
If we are getting this value from the hook, is it necessary to add a fallback here as the hook has a fallback?